### ISSUE TITLE
wc-generate: generate command fails due to node package dependency conflict

### ISSUE THREAD

**Issue by jason-capsule42** [AlaskaAirlines/WC-Generator#57]
## Describe the bug
wc-generate --name [name] command fails due node package dependency conflict. "rollup" and "rollup-plugin-minify-html-literals"

## To Reproduce
1. Run 'npm i @alaskaairux/wc-generator -g'
3. Run 'wc-generate --name counter'

`
Installing dependencies . . . . .Command failed: npm i
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! Found: rollup@2.38.0
npm ERR! node_modules/rollup
npm ERR!   dev rollup@"^2.38.0" from the root project
npm ERR!   peer rollup@"^1.20.0||^2.0.0" from @rollup/plugin-alias@3.1.1
npm ERR!   node_modules/@rollup/plugin-alias
npm ERR!     dev @rollup/plugin-alias@"^3.1.1" from the root project
npm ERR!   3 more (@rollup/plugin-babel, @rollup/plugin-commonjs, @rollup/plugin-node-resolve)
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! dev rollup-plugin-minify-html-literals@"^1.2.5" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: rollup@1.32.1
npm ERR! node_modules/rollup
npm ERR!   peer rollup@"^0.65.2 || ^1.0.0" from rollup-plugin-minify-html-literals@1.2.5
npm ERR!   node_modules/rollup-plugin-minify-html-literals
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/capsule/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/capsule/.npm/_logs/2021-01-24T17_31_26_910Z-debug.log
`

## Expected behavior
Expect no dependency issues that prevent npm install from running correctly

## Desktop (please complete the following information):
 - OS: macOS Big Sur
 - Version 11.1

**Comment by geoffrich** [comment]
I'm unable to repro on WSL. @blackfalcon are you able to repro?

```
➜ wc-generate --name test
? Did you review the Auro Governance Working Agreement? Yes
? Have you reviewed the Auro Components status board? Yes

Published: v2.9.4 | Installed: v2.9.4


 _____ _         _          _____ _     _ _
|  _  | |___ ___| |_ ___   |  _  |_|___| |_|___ ___ ___
|     | | .'|_ -| '_| .'|  |     | |  _| | |   | -_|_ -|
|__|__|_|__,|___|_,_|__,|  |__|__|_|_| |_|_|_|_|___|___|


 _ _ _ _____    _____                     _
| | | |     |  |   __|___ ___ ___ ___ ___| |_ ___ ___
| | | |   --|  |  |  | -_|   | -_|  _| .'|  _| . |  _|
|_____|_____|  |_____|___|_|_|___|_| |__,|_| |___|_|


Creating a Design System People Love.
    
Creating: /home/grich/git/generate-test/auro-test/.eslintrc.js
Creating: /home/grich/git/generate-test/auro-test/.github
Creating: /home/grich/git/generate-test/auro-test/.gitignore
Creating: /home/grich/git/generate-test/auro-test/.npmignore
Creating: /home/grich/git/generate-test/auro-test/.stylelintrc
Creating: /home/grich/git/generate-test/auro-test/.travis.yml
Creating: /home/grich/git/generate-test/auro-test/LICENSE
Creating: /home/grich/git/generate-test/auro-test/NOTICE
Creating: /home/grich/git/generate-test/auro-test/README.md
Creating: /home/grich/git/generate-test/auro-test/babel.config.js
Creating: /home/grich/git/generate-test/auro-test/demo
Creating: /home/grich/git/generate-test/auro-test/docs
Creating: /home/grich/git/generate-test/auro-test/index.html
Creating: /home/grich/git/generate-test/auro-test/index.js
Creating: /home/grich/git/generate-test/auro-test/karma.conf.js
Creating: /home/grich/git/generate-test/auro-test/package.json
Creating: /home/grich/git/generate-test/auro-test/packageScripts
Creating: /home/grich/git/generate-test/auro-test/rollup.config.js
Creating: /home/grich/git/generate-test/auro-test/scripts
Creating: /home/grich/git/generate-test/auro-test/src
Creating: /home/grich/git/generate-test/auro-test/test
Creating: /home/grich/git/generate-test/auro-test/packageScripts/postinstall.js
Creating: /home/grich/git/generate-test/auro-test/src/auro-test.js
Creating: /home/grich/git/generate-test/auro-test/src/es5.js
Creating: /home/grich/git/generate-test/auro-test/src/style.scss
Creating: /home/grich/git/generate-test/auro-test/test/auro-test.test.js
Creating: /home/grich/git/generate-test/auro-test/demo/alert.js
Creating: /home/grich/git/generate-test/auro-test/demo/demo.md
Creating: /home/grich/git/generate-test/auro-test/demo/index.html
Creating: /home/grich/git/generate-test/auro-test/demo/sass
Creating: /home/grich/git/generate-test/auro-test/.github/CODE_OF_CONDUCT.md
Creating: /home/grich/git/generate-test/auro-test/.github/CONTRIBUTING.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE
Creating: /home/grich/git/generate-test/auro-test/.github/PULL_REQUEST_TEMPLATE.md
Creating: /home/grich/git/generate-test/auro-test/.github/settings.yml
Creating: /home/grich/git/generate-test/auro-test/.github/stale.yml
Creating: /home/grich/git/generate-test/auro-test/.github/workflows
Creating: /home/grich/git/generate-test/auro-test/scripts/postCss.js
Creating: /home/grich/git/generate-test/auro-test/scripts/staticStyles-template.js
Creating: /home/grich/git/generate-test/auro-test/demo/sass/style.scss
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/audit.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/bug_report.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/config.yml
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/develop-design-story.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/feature_request.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/general-support.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/new_wc.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/parent-design-story.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/review-design-story.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/symbol-design-story.md
Creating: /home/grich/git/generate-test/auro-test/.github/ISSUE_TEMPLATE/user-story.md
Creating: /home/grich/git/generate-test/auro-test/.github/workflows/testPublish.yml
Creating: /home/grich/git/generate-test/auro-test/docs/index.html
Creating: /home/grich/git/generate-test/auro-test/docs/upstream.md

Copied all files!

Setting up Git .
Git repo successfully created

Renaming master to main .
Git branch successfully renamed

Installing dependencies . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Successfully installed dependencies!

Rebuilding node-sass . . . . . .
Successfully rebuilt node-sass!

Running initial component build . . . . . . . . . . . . . . . . . . . . . . . . . . .
Initial build success!

Committing generated resources . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . . .
Generated assets are committed!
Well done! The new HTML Custom Element auro-test has been created!
        
Dir: /home/grich/git/generate-test/auro-test
        
WC: v2.9.4
```

**Comment by blackfalcon** [comment]
Gonna have to chalk this up to an environmental issue that is unable to be reproduced.
